code/GridField/CopyButton.php

```diff
@@ -29,40 +38,59 @@ public function augmentColumns($gridField, &$columns)
 
     public function getColumnAttributes($gridField, $record, $columnName)
     {
-        return array('class' => 'col-buttons');
+        return ['class' => 'grid-field__col-compact'];
     }
 
     public function getColumnMetadata($gridField, $columnName)
     {
         if ($columnName == 'Actions') {
-            return array('title' => '');
+            return ['title' => ''];
         }
     }
 
     public function getColumnsHandled($gridField)
     {
-        return array('Actions');
+        return ['Actions'];
     }
 
     public function getActions($gridField)
     {
-        return array('copyrecord');
+        return ['copyrecord'];
     }
 
     public function getColumnContent($gridField, $record, $columnName)
     {
+        if($this->useAsColumn === false){
+            return;
+        }
+
         if (!$record->canCreate()) {
             return;
         }
-        $field = GridField_FormAction::create($gridField,  'CopyRecord'.$record->ID, false, "copyrecord",
-                array('RecordID' => $record->ID))
-            ->addExtraClass('gridfield-button-copy')
-            ->setAttribute('title', _t('GridAction.Copy', "Copy"))
-            ->setDescription(_t('GridAction.COPY_DESCRIPTION', 'Copy'));
+
+        $field = $this->getCopyAction($gridField, $record, $columnName);
 
         return $field->Field();
     }
 
+    private function getCopyAction($gridField, $record, $columnName)
+    {
+        $title = _t('GridAction.Copy', 'Copy');
+        $field = GridField_FormAction::create(
+            $gridField,
+            'CopyRecord'.$record->ID,
+            false,
+            "copyrecord",
+            ['RecordID' => $record->ID])
```

Comment:
this is some really weird code formatting. Can you place the `)` on the next line inline with the `$field` declaration and then move the subsequent arrow calls to be one tab from the `)`?